tests/e2e/tests/e2e_util/util.py

```diff
@@ -213,24 +208,24 @@ def setup_network(h: harness.Harness, instance_id: str):
 
 
 # Installs and setups the k8s snap on the given instance and connects the interfaces.
-def setup_k8s_snap(h: harness.Harness, instance_id: str, snap_path: Path):
+def setup_k8s_snap(i: harness.Instance, snap_path: Path):
     LOG.info("Install snap")
-    h.send_file(instance_id, config.SNAP, snap_path)
-    h.exec(instance_id, ["snap", "install", snap_path, "--dangerous"])
+    i.send_file(config.SNAP, snap_path)
+    i.exec(["snap", "install", snap_path, "--dangerous"])
 
     LOG.info("Initialize Kubernetes")
-    h.exec(instance_id, ["/snap/k8s/current/k8s/connect-interfaces.sh"])
+    i.exec(["/snap/k8s/current/k8s/connect-interfaces.sh"])
```

Comment:
nit: What do you think on `s/i/instance`? All the other methods use instance, and I am not very fond of single-char arguments.

Reply:
no complaints from me